st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -48,6 +52,17 @@ class ActionAliasController(resource.ContentPackResourceController):
         'sort': ['pack', 'name']
     }
 
+    _custom_actions = {
```

Comment:
Hm, it appears `_custom_actions` is only supported in newer versions of pecan because I get 404 in execute :/

We still use 0.7.0 because of backward incompatible changes in newer versions which make it hassle to upgrade.